metabase/metabase
# Add `PopoverBackButton` to Metabase UI

Extracts a commonly used component to `metabase/ui/buttons/PopoverBackButton` and replaces a couple of it's recent occurrences (in new `DatePicker` and `FilterPicker` components)

### How to verify

1. `yarn storybook`
2. Go to `https://localhost:6006/?path=/story/buttons-popoverbackbutton`
3. Click the button on the canvas, click "Nested item" to see the "Back" button

### Demo

<img alt="Back button demo" src="https://github.com/metabase/metabase/assets/17258145/17c34acb-b023-45d4-9435-0df0ffcb408b" width="380px" />
